### PR TITLE
updated label text to give more meaningful instructions

### PR DESCRIPTION
**Pull request by MelSumner**
If merged, this PR would update the text of label-type elements (buttons, values of aria-label, alt text on images, etc) to provide more meaningful content for users with assistive tech.

### CODE REVIEW

File: starter-files/starter.html [hunk 1]
```diff
@@ -189,15 +189,15 @@ <h1 id="message-label" class="sr-only">
         Message Input
       </h1>
 
-      <button class="text-3xl text-grey border-r-2 border-grey p-2" aria-label="File menu" type="button">
+      <button class="text-3xl text-grey border-r-2 border-grey p-2" aria-label="add a file to this message" type="button">
```
Comment (shankarsridhar) [inline starter-files/starter.html]: How about use the word `attach`, instead of `add` ?
`add a file to this message` => `attach a file to this message`

Comment (shankarsridhar) [inline starter-files/starter.html]: How about use the word `attach`, instead of `add` ?
`add a file to this message` => `attach a file to this message`

Comment (mike-north) [reply starter-files/starter.html]: I think either is fine

File: starter-files/starter.html [hunk 2]
```diff
@@ -189,15 +189,15 @@ <h5 class="text-sm">
         Message Input
       </p>
 
-      <button class="text-3xl text-grey border-r-2 border-grey p-2" aria-label="File menu" type="button">
+      <button class="text-3xl text-grey border-r-2 border-grey p-2" aria-label="add a file to this message">
```
Comment (mike-north) [inline starter-files/starter.html]: @MelSumner per our discussion this morning, I think it's safe to strip the `type="button"` I had here. Please let me know if I should put it back

Comment (MelSumner) [reply starter-files/starter.html]: Correct!

**Pull request merged**